core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -164,6 +169,30 @@ private SqlLibraryOperators() {
           ReturnTypes.VARCHAR_2000, null, OperandTypes.DATETIME,
           SqlFunctionCategory.TIMEDATE);
 
+  @LibraryOperator(libraries = {BIGQUERY, HIVE, SPARK})
+  public static final SqlFunction DATE_ADD =
+      new SqlFunction("DATE_ADD", SqlKind.PLUS,
+          ReturnTypes.DATE, null, OperandTypes.DATETIME,
+          SqlFunctionCategory.TIMEDATE) {
+
+        @Override public void unparse(SqlWriter writer, SqlCall call, int leftPrec, int rightPrec) {
+          writer.getDialect().unparseIntervalOperandsBasedFunctions(
+                writer, call, leftPrec, rightPrec);
+        }
+      };
+
+  @LibraryOperator(libraries = {HIVE, SPARK})
+  public static final SqlFunction ADD_MONTHS =
+      new SqlFunction("ADD_MONTHS", SqlKind.PLUS,
+          ReturnTypes.DATE, null, OperandTypes.DATETIME,
```

Comment:
please add numeric operand type as second argument.

Reply:
Added the needed types.